.github/workflows/deploy-unsigned.yml

```
@@ -108,25 +116,43 @@ jobs:
         env:
           GITHUB_ACCESS_TOKEN: ${{ secrets.ACCESS_TOKEN }}
           GITHUB_USERNAME: ${{ secrets.USERNAME }}
-        run: |
-          ./scripts/deploy-app.sh ${{ env.TARGET_ENV }} ${COMMIT_SHA:0:7}
+        run: ./scripts/deploy-app.sh ${{ env.TARGET_ENV }} ${COMMIT_SHA:0:7}
 
       - name: "Wait for LHDI deployment"
+        id: lhdi_deployment
```

Let's be consistent about naming:
```suggestion
        id: lhdi-deployment
```